lib/report-portal-client.js

```diff
@@ -28,6 +28,7 @@ class RPClient {
             headers: {
                 'User-Agent': 'NodeJS',
                 Authorization: `bearer ${params.token}`,
+                ...(params.headers || {}),
```

Comment:
Looks like `this.options` is not used anywhere. Maybe it should be in `this.headers`?

Reply:
Seems yes. Backporting issue. Will see

Reply:
fix it. squash commits, please, on merge.